lms/djangoapps/support/tests/test_views.py

```diff
@@ -1369,3 +1381,211 @@ def test_linking_program_enrollment_without_username(self, username, original_us
         response_data = json.loads(response.content.decode('utf-8'))
         error = "All linking lines must be in the format 'external_user_key,lms_username'"
         assert response_data['errors'] == [error]
+
+
+class TestOnboardingView(SupportViewTestCase, ProctoredExamTestCase):
+    """
+    Tests for OnboardingView
+    """
+
+    def setUp(self):
+        super().setUp()
+        SupportStaffRole().add_users(self.user)
+
+        self.proctored_exam_id = self._create_proctored_exam()
+        self.onboarding_exam_id = self._create_onboarding_exam()
+
+        self.other_user = User.objects.create(username='otheruser', password='test')
+        self.other_course_content = 'block-v1:test+course+2+type@sequential+block@other_onboard'
+
+        self.other_course = CourseFactory.create(
+            org='x',
+            course='y',
+            run='z',
```

Comment:
even if the data is dummy, use some meaningful words and avoid single characters.

Reply:
this is to align with exiting test cases for proctored exam.